bitcoin/src/amount.rs

```diff
@@ -1469,8 +1470,14 @@ mod verification {
         let n2 = kani::any::<i64>();
         kani::assume(n1.checked_add(n2).is_some()); // assume we don't overflow in the actual test
         kani::assume(n1.checked_sub(n2).is_some()); // assume we don't overflow in the actual test
-        assert_eq!(SignedAmount::from_sat(n1) + SignedAmount::from_sat(n2), SignedAmount::from_sat(n1 + n2));
-        assert_eq!(SignedAmount::from_sat(n1) - SignedAmount::from_sat(n2), SignedAmount::from_sat(n1 - n2));
+        assert_eq!(
+            SignedAmount::from_sat(n1) + SignedAmount::from_sat(n2),
+            SignedAmount::from_sat(n1 + n2)
+        );
+        assert_eq!(
+            SignedAmount::from_sat(n1) - SignedAmount::from_sat(n2),
+            SignedAmount::from_sat(n1 - n2)
+        );
```

Comment:
These are formatter changes that are unrelated to this PR. Since you have to rebase to fix CI can you remove these also please.

Reply:
Yeah, I'm trying to figure out how I messed up and broke CI. It looks completely unrelated to what I implemented and not sure how that got in the PR